<launch>
    <node pkg="pure_pursuit" name="pure_pursuit_controller" type="controller" respawn="false" output="screen">

        <param name="controller_freq" value="100" />
        <!--        speed -->
        <param name="speed_change" value="true" />
        <param name="base_speed" value="2.0" />
        <param name="min_speed" value="1.0" />
        <param name="max_speed" value="3.0" />
        <!--        acceleration -->
        <param name="acc_limit" value="0.3" />
        <param name="max_acc" value="2.0" />
        <param name="gas_weight" value="2.0" />
        <param name="back_weight" value="2.0" />
        <!-- lfw -->
        <param name="lfw_change" value="false" />
        <param name="base_lfw" value="0.6" />
        <param name="min_lfw" value="0.6" />
        <param name="max_lfw" value="1.2" />
        <remap from="/cmd_vel" to="/vesc_sim/cmd_vel" />

    </node>
</launch>